src/visibility-and-privacy.md

```diff
@@ -134,7 +137,6 @@ pub mod submodule {
         }
     }
 }
-
```

Comment:
This whitespace change seems unnecessary.  It would probably be best to keep consistency with the rest of the file.